idseq_dag/util/m8.py

```diff
@@ -262,8 +292,14 @@ def call_hit_level(hits, blacklisted = False):
                     # most specific taxonomy information.
                     emitted.add(read_id)
                     outf.write(line)
-                    msg = "{read_id}\t{hit_level}\t{taxid}\n".format(
-                        read_id=read_id, hit_level=hit_level, taxid=taxid)
+                    species_taxid = -1
+                    genus_taxid = -1
+                    family_taxid = -1
+                    if best_accession_id != None:
+                        (species_taxid, genus_taxid, family_taxid) = get_lineage(best_accession_id)
```

Comment:
`get_lineage` is very slow;  but in this case you are accessing a previous cached value, which should be fine.